java/client/src/main/java/glide/connectors/handlers/ChannelHandler.java

```diff
@@ -58,7 +58,7 @@ public ChannelHandler(
                         .channel(domainSocketChannelClass)
                         .handler(channelInitializer)
                         .connect(domainSocketAddress)
-                        // TODO call here .sync() if needed or remove this comment
+                        .syncUninterruptibly()
```

Comment:
is there a way to test this?

Reply:
An IT or a test with rust core mock can prove that it works fine.